pkg/controllers/user/alert/configsyncer/configsyncer.go

```diff
@@ -461,6 +462,17 @@ func (d *ConfigSyncer) addRecipients(notifiers []*v3.Notifier, receiver *alertco
 					ServiceKey:  alertconfig.Secret(notifier.Spec.PagerdutyConfig.ServiceKey),
 					Description: `{{ template "rancher.title" . }}`,
 				}
+
+				if notifier.Spec.PagerdutyConfig.HTTPClientConfig != nil {
+					url, err := toAlertManagerURL(notifier.Spec.PagerdutyConfig.HTTPClientConfig.ProxyURL)
+					if err != nil {
+						logrus.Errorf("Parse pageduty proxy url %s failed, %v", notifier.Spec.PagerdutyConfig.HTTPClientConfig.ProxyURL, err)
```

Comment:
nit: there's a typo in pagerduty spelling in this log (`pageduty`). Also can we change the error message to something like "Failed to parse pagerduty proxy url", for this and the other notifiers as well?

Reply:
Thanks, error message updated